src/index.js

```diff
@@ -116,43 +119,88 @@ const onMessageHandler = (target, context, message, fromSelf) => {
       })
       .catch(console.error);
   }
+  
+  if (msg === `${cmdPrefix}` + 'area') {
+    // a single area command - if the hawkuPath (in config.js) is populated, it runs like normal
+    // if there is an area manually declared, AND the path is populated, it will use the path
+    // if the area is declared, and the path is null, the static area will be used
+    if (config.commands.hawkuPath != null) {
+      const hawkuDetails = hawku.GetDetails();
+      const tabletArea = hawku.GetAreas();
+      let w = tabletArea.width; let h = tabletArea.height; let mw = tabletArea.maxwidth; let mh = tabletArea.maxheight;
+      let ar = hawkuDetails.forcedAspectRatio; let fullarea = hawkuDetails.fullArea;
+  
+      let areaMsg = function(width,height,maxwidth,maxheight,aspectratio,fullarea) {
```

Comment:
Some things: first, I think wrapping this in a function here is unnecessary.
```javascript
let message;
// message construction logic, assigns message = 'whatever'
client.say(target, message);
```
Second, use actual booleans here (I made a suggestion in hawku.js getDetails that should help with that), and reorder the logic to simplify things.
```javascript
if (fullArea) {
  if (forceAspectRatio) {
    message = 'whatever';
  } else { /* etc. */ }
} else {
  if (forceAspectRatio) {} else {}
}
```
Third, use template literals consistently here instead of sort of going half-and-half - it'll look cleaner.